src/sonic-eventd/rsyslog_plugin/timestamp_formatter.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
```

Comment:
Look at the solution here.

https://stackoverflow.com/questions/3786201/how-to-parse-date-time-from-string

Reply:
Unfortunately those solutions dont offer support for milliseconds, which is why I have kept regex